Makefile

```diff
@@ -23,8 +23,5 @@ clean:
 run: all
 	./$(BIN)/$(EXECUTABLE)
 
-# test:
-# 	$(BIN)/$(TEST)/$(EXECUTABLE): $(SRC)/*$(CC) $(C_FLAGS) -I$(INCLUDE) -L$(LIB) $^ -g -o $@ $(LIBRARIES)
-
```

Comment:
Thanks for cleaning this up!

Reply:
You cleaned this up! Haha, Mike thanking past Mike. Typical.